components/TableSection/TableSection.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, useState, useEffect } from "react";
+import styles from "./TableSection.module.css";
+
+import TableCell from "../TableCell/TableCell";
+
+interface TableSectionPropType {
+  tableData: any;
+  currentState: string;
+}
+
+const TableSection: FC<TableSectionPropType> = ({
+  tableData,
+  currentState,
+}) => {
+  return (
+    <>
+      <table className={styles.table_container}>
+        <thead>
+          <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Date</th>
+            <th scope="col">Description</th>
+            <th scope="col">Type</th>
+          </tr>
+        </thead>
+        <tbody className={styles.table_data}>
+          <TableCell typeName={""} />
```

Comment:
todo: Usually `TableCell` is the name given to the `td` but in this case you are using it for each `<tr>`. So a better name for this component is `<TabelRow>`

Reply:
Agree